docs/draft-release-notes.md

```diff
@@ -1,5 +1,9 @@
 ## Release notes for next release:
 
+### Breaking
```

Comment:
I will add this to my current open PR in docs https://github.com/pivotal-cf/docs-csb-aws/pull/64/files

Reply:
Thank you!